packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -166,6 +166,7 @@ export const PermissionsStruct = type({
     }),
   ),
   'endowment:webassembly': optional(object({})),
+  'endowment:user-input': optional(object({})),
```

Comment:
If we add this as a permission it should be added to: https://github.com/MetaMask/snaps/blob/main/packages/snaps-sdk/src/types/permissions.ts#L25